glide.lock

```diff
@@ -17,12 +17,17 @@ imports:
   version: 313961b101eb55f65ae0f03ddd4e322731763b6c
 - name: github.com/btcsuite/websocket
   version: 31079b6807923eb23992c421b114992b95131b55
+- name: github.com/davecgh/go-spew
```

Comment:
spew shouldn't have moved to imports, unless you regenerated the lockfile with some extra spew calls in non-test files. can you regen again?